src/posthog-core.js

```diff
@@ -135,6 +137,24 @@ var create_mplib = function (token, config, name) {
     instance['people'] = new PostHogPeople()
     instance['people']._init(instance)
 
+    // if any instance on the page has debug = true, we set the
+    // global debug to be true
+    Config.DEBUG = Config.DEBUG || instance.get_config('debug')
+
+    // if target is not defined, we called init after the lib already
+    // loaded, so there won't be an array of things to execute
+    if (!_.isUndefined(target) && _.isArray(target)) {
```

Comment:
Moving this code is very likely to break something - it has to do with support for if posthog-js is included multiple times and there are many different entrypoints to the script. I would err on being paranoid here - does this _need_ to be moved?